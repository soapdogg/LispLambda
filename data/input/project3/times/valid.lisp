(TIMES 3 34)
(TIMES (TIMES 1 2) (TIMES 2 (TIMES 5 6)))
(LESS (TIMES 12 32) (COND ((NULL (INT T)) 3)))