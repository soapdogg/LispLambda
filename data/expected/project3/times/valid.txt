102
120
NIL
